### PR TITLE
Autopilot as recommended way to operate a GKE cluster

### PR DESCRIPTION
**Pull request by danielmarzini**
Policy regarding Autopilot as recommended way to operate a GKE cluster

### CODE REVIEW

**Comment by danielmarzini** [comment]
@ewojtach would you please review this one? Thank you!

File: gke-policies/policy/autopilot_cluster.rego [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# METADATA
+# title: GKE Autopilot mode
+# description: GKE Autopilot mode is the recommended way to operate a GKE cluster
+# custom:
+#   group: Management
+#   severity: High
```
Comment (ewojtach) [inline gke-policies/policy/autopilot_cluster.rego:20]: should it be High severity?

**Review by ewojtach — APPROVED** [review]
LGTM!

File: gke-policies/policy/autopilot_cluster.rego (hunk 1 above)
Comment (danielmarzini) [reply gke-policies/policy/autopilot_cluster.rego:20]: agree that it seems to be too high, but it's now the recommended way to run GKE, wdyt? Maybe Medium it's a little bit less "Urgent"?

**Comment by mikouaj** [comment]
I would just add "recommendation" and "externalURI" to the custom metadata :)

**Review by mikouaj — APPROVED** [review]

**Pull request merged**